ext/ffi/ffi.stub.php

```diff
@@ -40,11 +40,11 @@ public static function sizeof(object $ptr): ?int {}
     public static function alignof(object $ptr): ?int {}
 
     /**
-     * @param FFI\CData|string $src
-     * @prefer-ref $dst
-     * @prefer-ref $src
+     * @param FFI\CData|string $from
+     * @prefer-ref $to
+     * @prefer-ref $from
      */
-    public static function memcpy(FFI\CData $dst, $src, int $size): void {}
+    public static function memcpy(FFI\CData $to, $from, int $size): void {}
```

Comment:
Using $from/$to with memcpy feels a bit odd to me, but it's not too bad...

Reply:
I'm ok with either :)